src/local_scheduler/local_scheduler_client.cc

```diff
@@ -9,6 +9,9 @@
 #include <sys/types.h>
 #include <unistd.h>
 
+// Using flatbuffer types definded in "format/local_scheduler_generated.h"
+using namespace ray::local_scheduler::protocol;
```

Comment:
Can we not use "using" and instead qualify uses? You can introduce an alias like `local_scheduler_protocol` as below.